pkg/kubelet/kubelet_node_status.go

```diff
@@ -137,10 +187,10 @@ func (kl *Kubelet) initialNodeStatus() (*api.Node, error) {
 
 	// @question: should this be place after the call to the cloud provider? which also applies labels
 	for k, v := range kl.nodeLabels {
-		if cv, found := node.ObjectMeta.Labels[k]; found {
+		if cv, found := node.Labels[k]; found {
```

Comment:
I realize there isn't an issue with the removal of ObjectMeta here, but if you want to reduce risk by reducing the # of LOC that change, you could omit these ObjectMeta changes.